stdlib/seq.mc

```diff
@@ -13,7 +13,7 @@ let init = lam seq. subsequence seq 0 (subi (length seq) 1)
 utest init [2,3,5] with [2,3]
 utest last [2,4,8] with 8
 
-let eqSeq = lam eq : (a -> a -> Bool). lam s1. lam s2.
+let eqSeq = lam eq : (a -> b -> Bool). lam s1. lam s2.
```

Comment:
The type should be `a -> a -> Bool` since `eqSet` is constrained by `eqMap` which requires maps with the same compare function (which constrains `eqSet` beyond the type on the equality function). Might be worth documenting this if you don't mind?